code/modules/tcg/cards.dm

```diff
@@ -66,6 +66,7 @@
 	desc = "A flipped TCG-branded card."
 	icon_state = "cardback"
 	icon = 'icons/obj/tcg/pack_1.dmi'
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
Cards already have 	w_class = WEIGHT_CLASS_TINY

Reply:
point where and i clear this out